src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -154,6 +155,8 @@ public partial interface IPublicClientApplication : IClientApplicationBase
         /// Flag to enable authentication with the user currently logeed-in in Windows.
         /// When set to true, the application will try to connect to the corporate network using windows integrated authentication.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("PublicClientApplication is now immutable, you can set this property using the PublicClientApplicationBuilder and read it using IAppConfig.  See https://aka.ms/msal-net-application-configuration", true)]
         bool UseCorporateNetwork { get; set; }
```

Comment:
should the setter also be marked as private? also should it be moved to migration aid?

Reply:
Don't need to set the setter to private since we will error on the Obsolete usage and it won't do anything.

---
In reply to: [250804050](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/833#discussion_r250804050) [](ancestors = 250804050)